docs/notebooks/visualizing_with_tensorboard.pct.py

```diff
@@ -0,0 +1,124 @@
+# %% [markdown]
+# # Tracking and visualizing optimizations using Tensorboard
+
+# %%
+import numpy as np
+import tensorflow as tf
+
+np.random.seed(1793)
+tf.random.set_seed(1793)
+
+# %% [markdown]
+# We often wish to track or visualize the Bayesian optimization process, either during and following execution. This tutorial shows how to do this using the [TensorBoard](https://www.tensorflow.org/tensorboard) visualization toolkit.
+
+# %% [markdown]
+# ## Set up the problem
+#
+# For this tutorial, we'll use the same set up as before.
+
+# %%
+import trieste
+import gpflow
+
+search_space = trieste.space.Box([0, 0], [1, 1])
+observer = trieste.objectives.utils.mk_observer(trieste.objectives.scaled_branin)
+initial_query_points = search_space.sample_sobol(5)
+initial_data = observer(initial_query_points)
+
+variance = tf.math.reduce_variance(initial_data.observations)
+kernel = gpflow.kernels.Matern52(variance=variance, lengthscales=[0.2, 0.2])
+gpr = gpflow.models.GPR(initial_data.astuple(), kernel, noise_variance=1e-5)
+gpflow.set_trainable(gpr.likelihood, False)
+model = trieste.models.gpflow.GaussianProcessRegression(gpr)
+
+# %% [markdown]
+# ## Setting up TensorBoard
+#
+# Before running the optimization loop, we decide where TensorBoard summary logs should be stored, and set a summary writer to do this.
+
+# %%
+# Clear any logs from previous runs
+# !rm -rf logs/tensorboard
+
+summary_writer = tf.summary.create_file_writer("logs/tensorboard")
```

Comment:
this would show how you can get automatic comparison of several different runs/experiments
```suggestion
summary_writer = tf.summary.create_file_writer("logs/tensorboard/exp1")
```